CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
```

Comment:
Minor comment, but we're trying to advocate just keeping all the contributing info in one place now, (e.g. https://github.com/screwdriver-cd/screwdriver/blob/master/CONTRIBUTING.md)

Reply:
Did you update the yo/generator

Reply:
Yes, it was updated in July
https://github.com/screwdriver-cd/generator-screwdriver/blob/master/app/templates/CONTRIBUTING.md

Reply:
Hmm, not sure why my old system didn't have the new version.  I just upgraded and it gave me the new one.   Updated file.